tests/lib/tools/tests.session

```diff
@@ -342,7 +342,7 @@ BEGIN {
 			Environment as 1 TERM=xterm-256color \
 			$selinux_context_arg \
 			ExecStart "a(sasb)" 1 \
-				"$(command -v runuser)" 6 "$(command -v runuser)" -l "$user" - -c "SNAPD_DEBUG=$SNAPD_DEBUG SNAP_CONFINE_DEBUG=$SNAP_CONFINE_DEBUG exec $tmp_dir/exec <$tmp_dir/stdin.pipe >$tmp_dir/stdout.pipe 2>$tmp_dir/stderr.pipe" false \
+				"$(command -v runuser)" 6 "$(command -v runuser)" -l "$user" - -c "SNAPD_DEBUG=$SNAPD_DEBUG SNAP_CONFINE_DEBUG=$SNAP_CONFINE_DEBUG PATH=$PATH:/snap/bin exec $tmp_dir/exec <$tmp_dir/stdin.pipe >$tmp_dir/stdout.pipe 2>$tmp_dir/stderr.pipe" false \
```

Comment:
Hm this will pull in the current user's (root) $PATH, perhaps we should just update the affected test to use snap run then? I took a quick look at the tests invokin `tests.session ... exec` and parallel-install-basic seems to be the only test that runs on core and does not use `snap run ..` to invoke the app.

Reply:
Thanks, that is a good idea. I was thinking about this but did not look what the other tests are doing, I think it's fine if all the other ones use snap run. I think it's very much a corner case created by tests.session anyway. I will do this and close this PR.